org.tahoma2d.Tahoma2D.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.Tahoma2D
+runtime: org.kde.Platform
+runtime-version: '23.08'
+sdk: "org.kde.Sdk" #Just pulling in the KDE stuff in case I truly need it.
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-download:rw
+  #Maybe add more later
+command: runtahoma2d
+#Luckily for us, Tahoma2D main dev manongjohn supplies a compiled appimage for us to use
```

Comment:
please build from source. opentoonz already does.

Reply:
OpenToonz doesn't officially provide a Linux version which is probably why it needed to be compiled here or is it policy you want all things to be compiled outright?

I provided an official Linux version of my app via AppImage.  Since it's already compiled on Ubuntu 20.04 through github, can we do something with an appimage-extract?   I know of another Linux-store maintainer that created a Debian package based on the appimage-extract.  I can't see why we can't do something similar here.

Please bear with me as I'm extremely new to setting up a flatpak. :)

Reply:
building from source is always preferred. Whether opentoonz provide binaries is irrelevant.